dist/src/main/java/io/camunda/application/commons/console/ping/PingConsoleRunner.java

```diff
@@ -138,5 +150,9 @@ public record ConsolePingConfiguration(
       String clusterId,
       String clusterName,
       Duration pingPeriod,
-      Map<String, String> properties) {}
+      RetryConfiguration retryConfiguration,
+      Map<String, String> properties) {
+    public record RetryConfiguration(
+        int numberOfMaxRetries, int retryDelayMultiplier, Duration maxRetryDelay) {}
```

Comment:
🔧 I'd adjust the naming slightly:
```yaml
console.ping:
  retry:
    maxAttempts:
    maxDelay:
    delayMultiplier:
```

Some justification:
1. `retryConfiguration` feels redundant, everything is "configuration" here
2. "retry" is already part of the prefix, no need to repeat it in every property
3. "numberOfMaxRetries" sounds like we only go up to the max a certain number of times, not that it's the max retry attempts.